src/com/generic/model/FreightType.java

```diff
@@ -0,0 +1,3 @@
+package com.generic.model;
+
+public enum FreightType {air, truck, ship, rail}
```

Comment:
enum entries should be capitalized. AIR, TRUCK, SHIP, RAIL

Reply:
Fixed